ansible/roles/linux-volumes-by-tag/files/openstack-disk-tag

```diff
@@ -2,12 +2,13 @@
 
 import argparse
 import contextlib
+import io
 import json
 import logging
-import pathlib
-import tempfile
 import subprocess
 
+import pycdlib
```

Comment:
nit: alphabetise imports unless there's a good reason that this one is out on its own?

Reply:
It is on it's own because I like to separate built-in imports from third-party ones. I know it isn't PEP8 but I also hate that.